Packages/com.chisel.editor/Chisel/Editor/Components/Containers/Model/Editor/CSGModelEditor.cs

```diff
@@ -11,9 +11,19 @@
 
 namespace Chisel.Editors
 {
+    public sealed class CSGModelDetails : ChiselNodeDetails<CSGModel>
+    {
+        const string ModelIconName = "csg_addition";
```

Comment:
Since we don't have an icon for the model, we use the "additive csg operation" icon for now 
(Packages\com.chisel.editor\Editor Resources\Icons\csg_addition.png on disk)